scripts/dkms.postbuild

```diff
@@ -12,8 +12,8 @@ while getopts "a:k:n:t:v:" opt; do
 	esac
 done
 
-if [ -z "${arch}" -o -z "${kver}" -o -z "${pkgname}" -o \
-     -z "${tree}" -o -z "${pkgver}" ]; then
+if [ -z "${arch}" ] || [ -z "${kver}" ] || [ -z "${pkgname}" ] || \
```

Comment:
I would prefer then "||" be inside the test rather than two separate tests, but I don't think it matters.